File: lcov/generator/src/second.js
function doNothing(arg) {
	if (typeof arg === 'number') {
		console.log('1...');
		console.log('2...');
		console.log('3...');
		console.log('4...');
		console.log('5...');
	}
	console.log('Ok');
}

module.exports = doNothing

File: jacoco/generator/src/main/java/fr/pinguet62/codacy/Second.java
package fr.pinguet62.codacy;

public class Second {

    static void doNothing(Object arg) {
        if (arg instanceof Number) {
            System.out.println("1...");
            System.out.println("2...");
            System.out.println("3...");
            System.out.println("4...");
            System.out.println("5...");
        }
        System.out.println("Ok");
    }

}
